extensions/2.0/Vendor/EXT_mesh_features/README.md

```diff
@@ -267,6 +271,32 @@ Empty feature IDs (e.g. `{}`) are disallowed — a feature ID must explicitly se
 > }
 > ```
 
+#### Resolving Feature IDs Externally
+
+Feature IDs do not have to be associated with a property table. It is also possible to specify feature IDs to only identify the features in an asset, and then use their IDs in an application-specific manner. 
+
+> **Example:** A minimal example for assigning feature IDs to a mesh primitive, using an implicit vertex attribute: The default value for the `offset` is 0, and the value for `repeat` is 3. This means that the vertices will receive the feature IDs `[0, 0, 0, 1, 1, 1, 2, 2, 2, ...]`, causing the vertices of each triangle to receive the same feature ID. 
+>
+> ```jsonc
+> // Primitive:
+> "extensions": {
+>   "EXT_mesh_features": {
+>     "featureIds": [
+>       { "repeat": 3 }
+>     ]
+>   }
+> }
+> ```
+
+These feature IDs can be used by the application in different ways. For example they could be used for styling features in a mesh, or to look up information for these features externally, via a REST API, or in a database.
```

Comment:
```suggestion
```

^Suggest merging this into the previous paragraph; see comment above.

Reply:
Indeed, the example did not add sooo much value here (and did not really "match" the diagram in any way, FWIW). 
I have removed it, and applied the other changes that you suggested.